anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/constant/ConstantThresholdModel.java

```diff
@@ -26,6 +27,6 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class ConstantThresholdModel implements AnomalyDetectorModel {
     private String uuid;
-    private String type;
+    private ModelTypeResource detectorType;
```

Comment:
`AnomalyDetectorModel` implementations should not have a `ModelTypeResource`. The implementation's class _is_ the type. In the database we need the type as a field since we're using a single model table to handle all types (i.e. it's one of the three standard strategies for modeling inheritance in a database), but the mapping to Java should resolve that "impedance mismatch".